CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# v2.2.1 (2024-07-24)
+
+## OS Changes
+
+* Update containerd to 1.7.2 ([#40])
```

Comment:
```suggestion
* Update containerd to 1.7.20 ([#40])
```

Reply:
Thanks - looking again I see I've missed the runc bump from that PR too, I'll push an udpate